build/build-validation.js

```diff
@@ -38,7 +38,7 @@ const defaultInitOptions = {
   onProtoPoisoning: 'error',
   onConstructorPoisoning: 'error',
   pluginTimeout: 10000,
-  requestIdHeader: 'request-id',
+  requestIdHeader: false,
```

Comment:
This would be a breaking change: we cannot ship it on v4

Reply:
Yes, so what would be the way to go then?

Reply:
We should create a `next` branch. IDK the status of it right now

Reply:
target is next branch